tests/helpers/test_update_coordinator.py

```diff
@@ -628,8 +628,7 @@ async def test_async_config_entry_first_refresh_invalid_state(
         RuntimeError,
         match="Detected code that uses `async_config_entry_first_refresh`, which "
         "is only supported when entry state is ConfigEntryState.SETUP_IN_PROGRESS, "
-        "but it is in state ConfigEntryState.NOT_LOADED. This will stop working "
-        "in Home Assistant 2025.11. Please report this issue",
+        "but it is in state ConfigEntryState.NOT_LOADED. Please report this issue",
```

Comment:
`with pytest.raises` => it's already stopped working so we no longer display the breaking version